python/src/deltachat/message.py

```diff
@@ -77,6 +77,23 @@ def set_text(self, text):
         """set text of this message. """
         lib.dc_msg_set_text(self._dc_msg, as_dc_charpointer(text))
 
+    @props.with_doc
+    def html(self):
+        """html text of this messages (might be empty if not an html message). """
+        return from_dc_charpointer(
+            lib.dc_get_msg_html(self.account._dc_context, self.id)) or ""
```

Comment:
here `Message.id` is used, since there is a bug in Python bindings wrt `id`, test will fail until #2326 is merged